### ISSUE TITLE
We're sorry, but something went wrong.

### ISSUE THREAD

**Issue by dutchakdev** [InfraexDev/peatio#8]
Hey! I write docker-compose.yml and start project:

```
version: "3"
services:
  dashm:
    build: ./
    volumes:
      - ./config/conf/rails-amqp.yml:/home/app/peatio/config/amqp.yml
      - ./config/conf/rails-database.yml:/home/app/peatio/config/database.yml
      - ./config/conf/rails-application.yml:/home/app/peatio/config/application.yml
      - ./config/conf/nginx-peatio-env.conf:/etc/nginx/main.d/peatio-env.conf
      - ./config/conf/nginx-peatio-with-ssl.conf:/etc/nginx/sites-available/peatio
    environment:
      RAILS_ENV: production
      DBMASTER_PORT_3306_TCP_ADDR: db
      DBMASTER_ENV_MYSQL_DATABASE: exchange
      DBMASTER_ENV_MYSQL_USER: root
      DBMASTER_ENV_MYSQL_PASSWORD: 'pass'
      REDIS_PORT: redis://redis:6379
      RABBIMQ_PORT_5672_TCP_PORT: 5672
      RABBIMQ_PORT_5672_TCP_ADDR: rabbitmq
    links:
      - db:db
      - redis:redis
      - rabbitmq:rebbitmq
    ports:
      - "8080:80"
    network_mode: bridge

  redis:
    image: redis
    restart: always
    network_mode: bridge

  rabbitmq:
    image: rabbitmq
    restart: always
    network_mode: bridge

  db:
    image: mariadb
    restart: always
    environment:
      MYSQL_ROOT_PASSWORD: 'pass'
    ports:
      - "13306:3306"
    network_mode: bridge
```

But have error `We're sorry, but something went wrong.`

Logs:

```
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
*** Running /etc/my_init.d/30_presetup_nginx.sh...
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 10
Feb  9 15:12:33 e3809d79aeef syslog-ng[23]: syslog-ng starting up; version='3.5.6'
ok: run: /etc/service/nginx-log-forwarder: (pid 29) 0s
nginx: [warn] invalid parameter "spdy": ngx_http_spdy_module was superseded by ngx_http_v2_module in /etc/nginx/sites-enabled/peatio:2
2018/02/09 15:12:34 [warn] 19#19: invalid parameter "spdy": ngx_http_spdy_module was superseded by ngx_http_v2_module in /etc/nginx/sites-enabled/peatio:2
[ 2018-02-09 15:12:34.7595 30/7f9a42ca3780 age/Wat/WatchdogMain.cpp:1283 ]: Starting Passenger watchdog...
[ 2018-02-09 15:12:34.7697 33/7fa068350780 age/Cor/CoreMain.cpp:1080 ]: Starting Passenger core...
[ 2018-02-09 15:12:34.7700 33/7fa068350780 age/Cor/CoreMain.cpp:248 ]: Passenger core running in multi-application mode.
[ 2018-02-09 15:12:34.7761 33/7fa068350780 age/Cor/CoreMain.cpp:830 ]: Passenger core online, PID 33
[ 2018-02-09 15:12:34.7895 39/7ffb4fd7c780 age/Ust/UstRouterMain.cpp:537 ]: Starting Passenger UstRouter...
[ 2018-02-09 15:12:34.7908 39/7ffb4fd7c780 age/Ust/UstRouterMain.cpp:350 ]: Passenger UstRouter online, PID 39
[ 2018-02-09 15:12:37.2806 33/7fa061714700 age/Cor/SecurityUpdateChecker.h:358 ]: A security update is available for your version (5.1.5) of Passenger, we strongly recommend upgrading to version 5.2.0.
[ 2018-02-09 15:12:37.2807 33/7fa061714700 age/Cor/SecurityUpdateChecker.h:363 ]:  Additional information:
- [Fixed in 5.1.6] [CVE-2017-7529] Specially crafted requests potentially trigger sensitive information leak via Nginx range module.
- [Fixed in 5.1.11] [URGENT] Users with write access to an application can view arbitrary files on the system if Passenger is running as root.
App 67 stdout:
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 60 was converted to "60". (StructuredWarnings::StandardWarning)
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. [] was converted to "[]". (StructuredWarnings::StandardWarning)
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 8080 was converted to "8080". (StructuredWarnings::StandardWarning)
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 471769 was converted to "471769". (StructuredWarnings::StandardWarning)
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 80 was converted to "80". (StructuredWarnings::StandardWarning)
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 80 was converted to "80". (StructuredWarnings::StandardWarning)
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 443 was converted to "443". (StructuredWarnings::StandardWarning)
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. true was converted to "true". (StructuredWarnings::StandardWarning)
App 67 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. false was converted to "false". (StructuredWarnings::StandardWarning)
App 67 stdout:
[ 2018-02-09 15:13:28.5235 33/7fa060205700 age/Cor/App/Implementation.cpp:304 ]: Could not spawn process for application /home/app/peatio: An error occurred while starting up the preloader.
  Error ID: 0e6d9581
  Error details saved to: /tmp/passenger-error-MGrkrd.html
  Message from application: No such file or directory @ rb_sysopen - /home/app/peatio/config/currencies.yml (Errno::ENOENT)
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/psych.rb:464:in `initialize'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/psych.rb:464:in `open'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/psych.rb:464:in `load_file'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/active_hash-1.5.3/lib/active_yaml/base.rb:22:in `load_path'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/active_hash-1.5.3/lib/active_file/hash_and_array_files.rb:7:in `raw_data'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/active_hash-1.5.3/lib/active_yaml/base.rb:9:in `load_file'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/active_hash-1.5.3/lib/active_file/base.rb:22:in `reload'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/active_hash-1.5.3/lib/active_file/base.rb:55:in `block (2 levels) in singleton class'
  /home/app/peatio/app/models/currency.rb:9:in `all'
  /home/app/peatio/app/api/api_v2/deposits.rb:12:in `block in <class:Deposits>'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/grape-0.7.0/lib/grape/validations.rb:97:in `instance_eval'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/grape-0.7.0/lib/grape/validations.rb:97:in `initialize'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/grape-0.7.0/lib/grape/validations.rb:290:in `new'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/grape-0.7.0/lib/grape/validations.rb:290:in `params'
  /home/app/peatio/app/api/api_v2/deposits.rb:10:in `<class:Deposits>'
  /home/app/peatio/app/api/api_v2/deposits.rb:4:in `<module:APIv2>'
  /home/app/peatio/app/api/api_v2/deposits.rb:3:in `<top (required)>'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/activesupport-4.0.13/lib/active_support/dependencies.rb:229:in `require'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/activesupport-4.0.13/lib/active_support/dependencies.rb:229:in `block in require'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/activesupport-4.0.13/lib/active_support/dependencies.rb:214:in `load_dependency'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/activesupport-4.0.13/lib/active_support/dependencies.rb:229:in `require'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/activesupport-4.0.13/lib/active_support/dependencies.rb:330:in `require_or_load'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/activesupport-4.0.13/lib/active_support/dependencies.rb:289:in `depend_on'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/activesupport-4.0.13/lib/active_support/dependencies.rb:207:in `require_dependency'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/engine.rb:465:in `block (2 levels) in eager_load!'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/engine.rb:464:in `each'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/engine.rb:464:in `block in eager_load!'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/engine.rb:462:in `each'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/engine.rb:462:in `eager_load!'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/engine.rb:347:in `eager_load!'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/application/finisher.rb:56:in `each'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/application/finisher.rb:56:in `block in <module:Finisher>'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/initializable.rb:30:in `instance_exec'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/initializable.rb:30:in `run'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/initializable.rb:55:in `block in run_initializers'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:226:in `block in tsort_each'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:348:in `block (2 levels) in each_strongly_connected_component'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:429:in `each_strongly_connected_component_from'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:347:in `block in each_strongly_connected_component'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:345:in `each'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:345:in `call'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:345:in `each_strongly_connected_component'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:224:in `tsort_each'
  /usr/local/rvm/rubies/ruby-2.2.5/lib/ruby/2.2.0/tsort.rb:203:in `tsort_each'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/initializable.rb:54:in `run_initializers'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/application.rb:215:in `initialize!'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/railties-4.0.13/lib/rails/railtie/configurable.rb:30:in `method_missing'
  /home/app/peatio/config/environment.rb:5:in `<top (required)>'
  config.ru:3:in `require'
  config.ru:3:in `block in <main>'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/rack-1.5.5/lib/rack/builder.rb:55:in `instance_eval'
  /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/rack-1.5.5/lib/rack/builder.rb:55:in `initialize'
  config.ru:1:in `new'
  config.ru:1:in `<main>'
  /usr/share/passenger/helper-scripts/rack-preloader.rb:110:in `eval'
  /usr/share/passenger/helper-scripts/rack-preloader.rb:110:in `preload_app'
  /usr/share/passenger/helper-scripts/rack-preloader.rb:156:in `<module:App>'
  /usr/share/passenger/helper-scripts/rack-preloader.rb:30:in `<module:PhusionPassenger>'
  /usr/share/passenger/helper-scripts/rack-preloader.rb:29:in `<main>'


[ 2018-02-09 15:13:28.5342 33/7fa060e92700 age/Cor/Con/CheckoutSession.cpp:285 ]: [Client 1-1] Cannot checkout session because a spawning error occurred. The identifier of the error is 0e6d9581. Please see earlier logs for details about the error.
App 168 stdout:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 60 was converted to "60". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. [] was converted to "[]". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 8080 was converted to "8080". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 471769 was converted to "471769". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 80 was converted to "80". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 80 was converted to "80". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 443 was converted to "443". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. true was converted to "true". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 168 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. false was converted to "false". (StructuredWarnings::StandardWarning)
App 168 stderr:
App 251 stdout:
Feb  9 15:17:01 e3809d79aeef CRON[1366]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
2018/02/09 15:27:33 [notice] 4803#4803: signal process started

[ 2018-02-09 15:27:33.7497 33/7fa060e92700 age/Cor/CoreMain.cpp:588 ]: Signal received. Gracefully shutting down... (send signal 2 more time(s) to force shutdown)
[ 2018-02-09 15:27:33.7529 33/7fa068350780 age/Cor/CoreMain.cpp:999 ]: Received command to shutdown gracefully. Waiting until all clients have disconnected...
[ 2018-02-09 15:27:33.7537 33/7fa068350780 age/Cor/CoreMain.cpp:917 ]: Checking whether to disconnect long-running connections for process 251, application /home/app/peatio/public
[ 2018-02-09 15:27:33.7567 33/7fa04b7fe700 Ser/Server.h:862 ]: [ServerThr.3] Freed 0 spare client objects
[ 2018-02-09 15:27:33.7567 33/7fa04affd700 Ser/Server.h:862 ]: [ServerThr.4] Freed 0 spare client objects
[ 2018-02-09 15:27:33.7567 33/7fa04bfff700 Ser/Server.h:862 ]: [ServerThr.2] Freed 0 spare client objects
[ 2018-02-09 15:27:33.7567 33/7fa04b7fe700 Ser/Server.h:490 ]: [ServerThr.3] Shutdown finished
[ 2018-02-09 15:27:33.7567 33/7fa04affd700 Ser/Server.h:490 ]: [ServerThr.4] Shutdown finished
[ 2018-02-09 15:27:33.7567 33/7fa04bfff700 Ser/Server.h:490 ]: [ServerThr.2] Shutdown finished
[ 2018-02-09 15:27:33.7568 33/7fa060e92700 Ser/Server.h:862 ]: [ServerThr.1] Freed 0 spare client objects
[ 2018-02-09 15:27:33.7568 33/7fa060e92700 Ser/Server.h:490 ]: [ServerThr.1] Shutdown finished
[ 2018-02-09 15:27:33.7576 33/7fa04a7fc700 Ser/Server.h:862 ]: [ApiServer] Freed 0 spare client objects
[ 2018-02-09 15:27:33.7588 33/7fa04a7fc700 Ser/Server.h:490 ]: [ApiServer] Shutdown finished
[ 2018-02-09 15:27:33.7598 33/7fa068350780 age/Cor/CoreMain.cpp:917 ]: Checking whether to disconnect long-running connections for process 251, application /home/app/peatio/public

[ 2018-02-09 15:27:33.7713 39/7ffb49140700 age/Ust/UstRouterMain.cpp:430 ]: Signal received. Gracefully shutting down... (send signal 2 more time(s) to force shutdown)
[ 2018-02-09 15:27:33.7714 39/7ffb4fd7c780 age/Ust/UstRouterMain.cpp:500 ]: Received command to shutdown gracefully. Waiting until all clients have disconnected...
[ 2018-02-09 15:27:33.7746 39/7ffb4893f700 Ser/Server.h:862 ]: [UstRouterApiServer] Freed 0 spare client objects
[ 2018-02-09 15:27:33.7755 39/7ffb4893f700 Ser/Server.h:490 ]: [UstRouterApiServer] Shutdown finished
[ 2018-02-09 15:27:33.7747 39/7ffb49140700 Ser/Server.h:490 ]: [UstRouter] Shutdown finished
[ 2018-02-09 15:27:33.7809 39/7ffb4fd7c780 age/Ust/UstRouterMain.cpp:531 ]: Passenger UstRouter shutdown finished
[ 2018-02-09 15:27:33.9991 4806/7f34d5b87780 age/Wat/WatchdogMain.cpp:1283 ]: Starting Passenger watchdog...
[ 2018-02-09 15:27:34.0130 4809/7f07842d9780 age/Cor/CoreMain.cpp:1080 ]: Starting Passenger core...
[ 2018-02-09 15:27:34.0132 4809/7f07842d9780 age/Cor/CoreMain.cpp:248 ]: Passenger core running in multi-application mode.
[ 2018-02-09 15:27:34.0219 4809/7f07842d9780 age/Cor/CoreMain.cpp:830 ]: Passenger core online, PID 4809
[ 2018-02-09 15:27:34.0337 4817/7fca6fd37780 age/Ust/UstRouterMain.cpp:537 ]: Starting Passenger UstRouter...
[ 2018-02-09 15:27:34.0349 4817/7fca6fd37780 age/Ust/UstRouterMain.cpp:350 ]: Passenger UstRouter online, PID 4817
[ 2018-02-09 15:27:36.0620 33/7fa068350780 age/Cor/CoreMain.cpp:1065 ]: Passenger core shutdown finished
App 4859 stdout:
[ 2018-02-09 15:27:36.4958 4809/7f077d69d700 age/Cor/SecurityUpdateChecker.h:358 ]: A security update is available for your version (5.1.5) of Passenger, we strongly recommend upgrading to version 5.2.0.
[ 2018-02-09 15:27:36.4958 4809/7f077d69d700 age/Cor/SecurityUpdateChecker.h:363 ]:  Additional information:
- [Fixed in 5.1.6] [CVE-2017-7529] Specially crafted requests potentially trigger sensitive information leak via Nginx range module.
- [Fixed in 5.1.11] [URGENT] Users with write access to an application can view arbitrary files on the system if Passenger is running as root.
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 60 was converted to "60". (StructuredWarnings::StandardWarning)
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. [] was converted to "[]". (StructuredWarnings::StandardWarning)
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 8080 was converted to "8080". (StructuredWarnings::StandardWarning)
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 471769 was converted to "471769". (StructuredWarnings::StandardWarning)
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 80 was converted to "80". (StructuredWarnings::StandardWarning)
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 80 was converted to "80". (StructuredWarnings::StandardWarning)
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 443 was converted to "443". (StructuredWarnings::StandardWarning)
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. true was converted to "true". (StructuredWarnings::StandardWarning)
App 4859 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. false was converted to "false". (StructuredWarnings::StandardWarning)
App 4978 stdout:
*** Shutting down runit daemon (PID 10)...
*** Killing all processes...
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
*** Running /etc/my_init.d/30_presetup_nginx.sh...
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 9
ok: run: /etc/service/nginx-log-forwarder: (pid 19) 0s
[ 2018-02-09 15:49:03.9730 4809/7f07667fc700 Ser/Server.h:490 ]: [ApiServer] Shutdown finished
[ 2018-02-09 15:49:03.9731 4809/7f07842d9780 age/Cor/CoreMain.cpp:917 ]: Checking whether to disconnect long-running connections for process 4978, application /home/app/peatio/public
[ 2018-02-09 15:49:03.9735 4817/7fca6fd37780 age/Ust/UstRouterMain.cpp:531 ]: Passenger UstRouter shutdown finished
[ 2018-02-09 15:49:04.2467 4809/7f07842d9780 age/Cor/CoreMain.cpp:1065 ]: Passenger core shutdown finished
[ 2018-02-09 15:49:11.8852 27/7f013cd31780 age/Wat/WatchdogMain.cpp:1283 ]: Starting Passenger watchdog...
[ 2018-02-09 15:49:11.9008 30/7f5a1d350780 age/Cor/CoreMain.cpp:1080 ]: Starting Passenger core...
[ 2018-02-09 15:49:11.9011 30/7f5a1d350780 age/Cor/CoreMain.cpp:248 ]: Passenger core running in multi-application mode.
[ 2018-02-09 15:49:11.9081 30/7f5a1d350780 age/Cor/CoreMain.cpp:830 ]: Passenger core online, PID 30
[ 2018-02-09 15:49:11.9262 36/7faa37b07780 age/Ust/UstRouterMain.cpp:537 ]: Starting Passenger UstRouter...
[ 2018-02-09 15:49:11.9276 36/7faa37b07780 age/Ust/UstRouterMain.cpp:350 ]: Passenger UstRouter online, PID 36
Feb  9 15:49:11 e3809d79aeef syslog-ng[20]: syslog-ng starting up; version='3.5.6'
[ 2018-02-09 15:49:14.3686 30/7f5a16714700 age/Cor/SecurityUpdateChecker.h:358 ]: A security update is available for your version (5.1.5) of Passenger, we strongly recommend upgrading to version 5.2.0.
[ 2018-02-09 15:49:14.3686 30/7f5a16714700 age/Cor/SecurityUpdateChecker.h:363 ]:  Additional information:
- [Fixed in 5.1.6] [CVE-2017-7529] Specially crafted requests potentially trigger sensitive information leak via Nginx range module.
- [Fixed in 5.1.11] [URGENT] Users with write access to an application can view arbitrary files on the system if Passenger is running as root.
```

**Comment by muhammednagy** [comment]
Message from application: No such file or directory @ rb_sysopen - /home/app/peatio/config/currencies.yml (Errno::ENOENT)
you need currencies.yml 
If it all worked make a pull request

**Comment by dutchakdev** [comment]
Yes, i will make, but when all finish. Old docker sh scripts is bad way and needed refactoring

**Comment by dutchakdev** [comment]
I fix it but have this error:

```
dashm_1     | *** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
dashm_1     | *** Running /etc/my_init.d/30_presetup_nginx.sh...
dashm_1     | *** Running /etc/rc.local...
dashm_1     | *** Booting runit daemon...
dashm_1     | *** Runit started as PID 10
dashm_1     | ok: run: /etc/service/nginx-log-forwarder: (pid 21) 0s
dashm_1     | App 65 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 443 was converted to "443". (StructuredWarnings::StandardWarning)
dashm_1     | App 65 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. true was converted to "true". (StructuredWarnings::StandardWarning)
dashm_1     | App 65 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. false was converted to "false". (StructuredWarnings::StandardWarning)
dashm_1     | App 148 stdout:
dashm_1     | [ 2018-02-09 18:02:07.5338 28/7f7e81fe5780 age/Wat/WatchdogMain.cpp:1283 ]: Starting Passenger watchdog...
dashm_1     | [ 2018-02-09 18:02:07.5506 31/7f4c02ed1780 age/Cor/CoreMain.cpp:1080 ]: Starting Passenger core...
dashm_1     | [ 2018-02-09 18:02:07.5512 31/7f4c02ed1780 age/Cor/CoreMain.cpp:248 ]: Passenger core running in multi-application mode.
dashm_1     | [ 2018-02-09 18:02:07.5640 31/7f4c02ed1780 age/Cor/CoreMain.cpp:830 ]: Passenger core online, PID 31
dashm_1     | [ 2018-02-09 18:02:07.5740 37/7f4e75f61780 age/Ust/UstRouterMain.cpp:537 ]: Starting Passenger UstRouter...
dashm_1     | [ 2018-02-09 18:02:07.5781 37/7f4e75f61780 age/Ust/UstRouterMain.cpp:350 ]: Passenger UstRouter online, PID 37
dashm_1     | Feb  9 18:02:07 e3809d79aeef syslog-ng[18]: syslog-ng starting up; version='3.5.6'
dashm_1     | [ 2018-02-09 18:02:09.9428 31/7f4bfc295700 age/Cor/SecurityUpdateChecker.h:358 ]: A security update is available for your version (5.1.5) of Passenger, we strongly recommend upgrading to version 5.2.0.
dashm_1     | [ 2018-02-09 18:02:09.9429 31/7f4bfc295700 age/Cor/SecurityUpdateChecker.h:363 ]:  Additional information:
dashm_1     | - [Fixed in 5.1.6] [CVE-2017-7529] Specially crafted requests potentially trigger sensitive information leak via Nginx range module.
dashm_1     | - [Fixed in 5.1.11] [URGENT] Users with write access to an application can view arbitrary files on the system if Passenger is running as root.
dashm_1     | App 83 stdout:
dashm_1     | App 83 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. [] was converted to "[]". (StructuredWarnings::StandardWarning)
dashm_1     | App 83 stderr:
dashm_1     | App 83 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 8080 was converted to "8080". (StructuredWarnings::StandardWarning)
dashm_1     | App 83 stderr:
dashm_1     | App 83 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 471769 was converted to "471769". (StructuredWarnings::StandardWarning)
dashm_1     | App 83 stderr:
dashm_1     | App 83 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 80 was converted to "80". (StructuredWarnings::StandardWarning)
dashm_1     | App 83 stderr:
dashm_1     | App 83 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 80 was converted to "80". (StructuredWarnings::StandardWarning)
dashm_1     | App 83 stderr:
dashm_1     | App 83 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. 443 was converted to "443". (StructuredWarnings::StandardWarning)
dashm_1     | App 83 stderr:
dashm_1     | App 83 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. true was converted to "true". (StructuredWarnings::StandardWarning)
dashm_1     | App 83 stderr:
dashm_1     | App 83 stderr: /home/app/peatio/vendor/bundle/ruby/2.2.0/gems/figaro-1.1.1/lib/figaro/application.rb:84:in `non_string_configuration!' : WARNING: Use strings for Figaro configuration. false was converted to "false". (StructuredWarnings::StandardWarning)
dashm_1     | App 83 stderr:
dashm_1     | App 166 stdout:
dashm_1     | Feb  9 18:17:01 e3809d79aeef CRON[180]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)

``

**Comment by dutchakdev** [comment]
i fix it, read logs/production.log and saw 

```
ArgumentError (Invalid Timezone: YOUR LOCALE):
  app/controllers/application_controller.rb:112:in `set_timezone'
  lib/middleware/security.rb:11:in `call'
```

Fixed.
